allennlp/training/trainer.py

```diff
@@ -105,13 +104,29 @@ def add_train_histogram(self, name: str, values: torch.Tensor, global_step: int)
             if isinstance(values, torch.autograd.Variable):
                 values_to_write = values.cpu().data.numpy().flatten()
                 self._train_log.file_writer.add_summary(
-                        tb_histogram(name, values_to_write), global_step
+                        tensorboard.summary.histogram(name, values_to_write), global_step
                 )
 
     def add_validation_scalar(self, name: str, value: float, global_step: int) -> None:
         if self._validation_log is not None:
             self._validation_log.add_scalar(name, value, global_step)
 
+def time_to_str(timestamp: int) -> str:
+    """
+    Convert seconds past Epoch to human readable string.
+    """
+    datetimestamp = datetime.datetime.fromtimestamp(timestamp)
+    return '{:04d}-{:02d}-{:02d}-{:02d}-{:02d}-{:02d}'.format(
+            datetimestamp.year, datetimestamp.month, datetimestamp.day,
+            datetimestamp.hour, datetimestamp.minute, datetimestamp.second
+    )
+
+def str_to_time(time_str: str) -> datetime.datetime:
+    """
+    Convert human readable string to datetime.datetime.
+    """
+    pieces: Any = [int(piece) for piece in time_str.split('-')]
```

Comment:
Why is this `Any`, it should be `List[int]`.

Reply:
This was to get mypy to pass.  It really didn't like my `*pieces` on the next line.